### ISSUE TITLE
iebaltab : append tables

### ISSUE THREAD

**Issue by kbjarkefur** [worldbank/ietoolkit#98]
Allow appending tables exported from **iebaltab** the way, for example, **estout** does it.

Is that something we can implement? Working with and modifying different file formats can be tricky, but if someone sees a straightforward way to implement this, then I would be happy to include it.

I guess it would only apply to tables exported to Excel. Or could this make sense in LaTeX too?

Based on feedback from @aidancoville.

**Comment by luizaandrade** [comment]
I think it would make sense for LaTeX too. Michael asked me something similar yesterday, though not for iebaltab. I have something in mind, but have to check if it works.

**Comment by morevba** [comment]
Yes, this would be ideal. I am using estout to append table and exporting that to Latex which is tricky but works fine. Being able to do that with iebaltab would be great for future reports.

**Comment by kbjarkefur** [comment]
Ok, great! Let's look into it!

My biggest concern for both Excel and LaTeX would be how to deal with titles and notes. If they are the same across all appended tables, then no problem. But if it is not, it is not obvious immediately to me 1) how to test that it is similar, and 2) if we should repeat the rows with titles and notes for each table or how to deal with that.
